### DISCUSSION TITLE
Soloqueue for Arena 3v3 and/or 2v2

### DISCUSSION THREAD

**Discussion by Tov23** [chromiecraft/chromiecraft#6263] [Suggestions]
This feature has always been desirable to PVP players, but also to casual PVE players who did not mind dabbing into PVE if it meant it was less important to find the right team composition.

Also, the new adversary server that shall not be named has launched with this feature and I fear that it might attract the PVP community from here, which is already not that big.

You guys stand out with amazing features, I'm sure this one would only benefit the community. It might attract new players as well.
Keep up to the good work that you're doing!

**Comment by Helias** [comment]
We released 1v1 and 3v3soloQ, so we can close this discussion.

We'll keep in mind the possibility of introducing 2v2soloQ and/or 5v5soloQ, thanks for the discussion.

Release article video and reference:
[youtube.com/watch?v=P7hYa2rwWqo](https://www.youtube.com/watch?v=P7hYa2rwWqo)
[chromiecraft.com/en/pvp-revamp-arena-s3](https://www.chromiecraft.com/en/pvp-revamp-arena-s3/)
